### PR TITLE
filtering double quotes on CMD's output

### PR DESCRIPTION
**Pull request by cordoval**
|Q            |A  |
|---          |---|
|Fixed Tickets|   |
|License      |MIT|

### CODE REVIEW

File: src/Helper/ProcessHelper.php [hunk 1]
```diff
@@ -80,7 +80,10 @@ public function runCommand($command, $allowFailures = false, $callback = null, $
         $process = $builder->getProcess();
 
         $remover = function ($untrimmed) {
-            return ltrim(rtrim($untrimmed, "'"), "'");
+            $removedSingleQuotes = ltrim(rtrim($untrimmed, "'"), "'");
+            $removedDoubleuotes = ltrim(rtrim($removedSingleQuotes, '"'), '"');
```
Comment (pierredup) [inline src/Helper/ProcessHelper.php]: Wouldn't just `trim($untrimmed, '\'"')` be enough?

Comment (cordoval) [inline src/Helper/ProcessHelper.php]: yeah could be, that is a better way to put it

Comment (jacobkiers) [inline src/Helper/ProcessHelper.php]: Typo: should be `$removedDoubleQuotes`.

Comment (cordoval) [inline src/Helper/ProcessHelper.php]: @sstok should we then not do this change? since you say that it is how it is passed to the command?

Comment (sstok) [inline src/Helper/ProcessHelper.php]: Its better to leave the output as it was executed. Its meanly for debugging so normally you would not see it :)

**Pull request closed without merging**